### PR TITLE
SUBMARINE-666. Update HowToRelease.md

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?
update HowToRelease.md, make it more clear and fix some typos

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-666

### How should this be tested?
https://travis-ci.org/github/pingsutw/hadoop-submarine/builds/742603121

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: dev-support/cicd/HowToRelease.md [hunk 1]
```diff
@@ -26,16 +29,16 @@ svn ci -m "Add <your name>'s key"
 It's better to send a release plan email informing code freeze date and release date.
 
 ## 3. Clean up the JIRA
-Bulk update JIRA to unassign from this release all issues that are open non-blockers.
+Bulk update JIRA to unassigned from this release all issues that are open non-blockers.
```
Comment (jojochuang) [inline dev-support/cicd/HowToRelease.md:32]: Let's update this to "Bulk update JIRA to move out non-blocker issues by setting the target version to the new release."

File: dev-support/cicd/HowToRelease.md [hunk 2]
```diff
@@ -95,7 +98,7 @@ docker tag local/mini-submarine:0.3.0 apache/submarine:mini-0.3.0-RC0
 ```
 In the container, we can verify that the submarine jar version is the expected 0.3.0. Then we can upload this image with a "RC" tag for a vote.
 ```
-docker push apache/mini-submarine:0.3.0:RC0
+docker push apache/submarine:mini-0.3.0-RC0
```
Comment (jojochuang) [inline dev-support/cicd/HowToRelease.md:101]: Does it require an INFRA ticket to grant you permission to the ASF DockerHub? like this one? https://issues.apache.org/jira/browse/INFRA-20364

**Review by xunliu — APPROVED** [review]
Will merge if no more comments.

**Review by jojochuang — APPROVED** [review]
+1

**Pull request closed without merging**